Sim_Harness_GUI/MainWindow.cs

```diff
@@ -149,17 +162,58 @@ private bool checkFiles()
 	private string buildStartString()
 	{
 		string jsonString = "{\n\t\"TimeFrame\": {" +
-			"\n\t\t\"wall\": \"" + DateTime.Now.ToString("o") + "\"," +
-			"\n\t\t\"sim\": \"" + DateTime.Now.ToString("o") + "\"," + 
+			"\n\t\t\"wall\": \"" + DateTime.Now.ToString("O") + "\"," +
+			"\n\t\t\"sim\": \"" + _simTime.ToString("O") + "\"," +
 			"\n\t\t\"rate\": " + timeFrameSpeedSpinbutton.Text +
```

Comment:
You realize this entire statement is solved much more safely by simply using `JsonConvert.SerializeObject()`? I could've sworn I made this change awhile back.